.travis.yml

```diff
@@ -37,15 +37,8 @@ after_success:
   - bash ./pr_deploy.sh
   - bash ./deploy.sh
 
-deploy:
```

Comment:
Why removing this deployment? Is it not working?

Reply:
It doesn't seems to work. The deployed link not working since it is not able to find the build files. We need to write a separate shell script and call it on ```after_success```.